Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/TimeView/TimeViewPlugin.cpp

```diff
@@ -2036,6 +2072,24 @@ namespace EMStudio
         GetMainWindow()->OnUpdateRenderPlugins();
     }
 
+    void TimeViewPlugin::toggleMotionEventPresetsPane()
+    {
+        QList<int> sizes = m_paneSplitter->sizes();
+        if (sizes[1] == 0)
+        {
+            sizes[1] = m_motionEventsPresetsWidget->minimumSizeHint().width();
+            sizes[0] = m_paneSplitter->width() - sizes[1];
+        }
```

Comment:
Is the splitter grab- and resizeable as well?

Reply:
yes, it's freely resizeable. this is just for the toggle button above